src/Adapter/MSTest.TestAdapter/PublicAPI/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,3 @@
 ﻿#nullable enable
+Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter.Execution.TestAssemblyInfo.AssemblyCleanupException.get -> System.Exception?
```

Comment:
Can we avoid having these public?

Reply:
It can be internal. `AssemblyInitalizeException` is public. So, I followed the same pattern.

Reply:
Not sure why the other is public, let's avoid making things public when there is no strong need for it.

Reply:
updated,